ecms_base/config/install/user.role.authenticated.yml

```diff
@@ -6,6 +6,15 @@ label: 'Authenticated user'
 weight: 1
 is_admin: false
 permissions:
+  - 'access administration pages'
```

Comment:
Are there any security implications here by giving the authenticated user these administrative permissions? Should we move these to the elevated author/editor roles instead? I _think_ it is fine, but wanted to flag it anyway.

Reply:
I will test how removing this impacts the content author role and adjust accordingly